packages/preview-button/admin/src/components/PreviewLink/index.js

```diff
@@ -76,6 +78,29 @@ const PreviewLink = () => {
   const additionalInformationUid =
     isAdditionalInformation && data.initialData?.additional_information[0]?.href.replace(/\/\d+$/, '');
   const additionalInformationId = isAdditionalInformation && data.initialData?.additional_information[0]?.id;
+  const productInternalFieldBlock =
+    Array.isArray(data.initialData?.sections) &&
+    data.initialData?.sections?.find((section) => section?.__component === 'components.internal-block-content')
+      ?.internal_field;
+  const productInternalFieldBlockUID = productInternalFieldBlock?.[0]?.href?.replace(/\/\d+$/, '');
```

Comment:
Misschien kun je deze regexp replace in een kleine (lokale) util functie neerzetten met een zelf-documenterende naam?